libfreefare/freefare.h

```diff
@@ -540,7 +540,10 @@ typedef enum mifare_key_type {
 struct mifare_key_deriver;
 typedef struct mifare_key_deriver *MifareKeyDeriver;
 
-MifareKeyDeriver mifare_key_deriver_new_an10922(MifareDESFireKey master_key, MifareKeyType output_key_type);
+#define AN10922_FLAG_DEFAULT            0
+#define AN10922_FLAG_EMULATE_ISSUE_91   (1<<1)
```

Comment:
What about using an enum here?  That way we can use a switch statement to choose from an implementation or another (more readable than defines imho).

Reply:
Also, there was no release of the library with broken code, and the version is still 0.xxx so I do think that keeping backward compatibility with buggy features is a must-have.  If you think this can be simplified by only providing a correct implementation, please go this way!

Reply:
Ah okay, I was precisely suggesting an enum to avoid mixing the values, but if it's on purpose, then go for it :+1: